src/render/OpenGL.cpp

```diff
@@ -1124,7 +1124,7 @@ void CHyprOpenGLImpl::renderSplash(cairo_t *const CAIRO, cairo_surface_t *const
     cairo_text_extents_t textExtents;
     cairo_text_extents(CAIRO, g_pCompositor->m_szCurrentSplash.c_str(), &textExtents);
 
-    cairo_move_to(CAIRO, m_RenderData.pMonitor->vecPixelSize.x / 2.f - textExtents.width / 2.f, m_RenderData.pMonitor->vecPixelSize.y - textExtents.height - 1);
```

Comment:
wot

Reply:
should i take that out? It was the result of commit to test the gh token i just made.
Didn't know that it will change the pull request.
Shouldn't change the behavior, and if unoptimized, it should be a bit more efficient.

Reply:
I changed the 2.f to 2.0 since all the values are doubles and cairo_move_to takes doubles too.

Reply:
By the way, am i supposed to resolve a conversation like this?

Reply:
remove this change. it's unrelated to the pr